nixos/modules/programs/ssh.nix

```
@@ -305,11 +303,8 @@ in
         AddressFamily ${if config.networking.enableIPv6 then "any" else "inet"}
         GlobalKnownHostsFile ${concatStringsSep " " knownHostsFiles}
 
-        ${optionalString cfg.setXAuthLocation ''
-          XAuthLocation ${pkgs.xorg.xauth}/bin/xauth
-        ''}
-
-        ForwardX11 ${if cfg.forwardX11 then "yes" else "no"}
+        ${optionalString cfg.setXAuthLocation "XAuthLocation ${pkgs.xorg.xauth}/bin/xauth"}
+        ${lib.optionalString (cfg.forwardX11 != null) "ForwardX11 ${if cfg.forwardX11 then "yes" else "no"}"}
```

I'd prefer not touching XAuthLocation unnecessarily, as reformatting is the bane of `git blame`.

```suggestion
        ${optionalString cfg.setXAuthLocation ''
          XAuthLocation ${pkgs.xorg.xauth}/bin/xauth
        ''}
        ${lib.optionalString (cfg.forwardX11 != null) ''
          ForwardX11 ${if cfg.forwardX11 then "yes" else "no"}
        ''}
```

When templating a config file i believe multiline formatting is better anyway